src/interface/sparse_variable.hpp

```diff
@@ -99,6 +99,9 @@ class SparseVariable {
 
   bool IsSet(const MetadataFlag flag) { return metadata_.IsSet(flag); }
 
+  inline bool HasFluxes() const { return metadata_.IsSet(Metadata::WithFluxes); }
+  inline bool HasBoundaryVars() const { return metadata_.IsSet(Metadata::FillGhost); }
```

Comment:
Hmm now that I'm seeing these in action, I'm not sure I like them. This feels like it obscures the meaning of `WithFluxes` and `FillGhost` more than it helps... now it's harder to grep through the code to find where metadata is used. I think I prefer just making the `IsSet` calls... But I don't feel strongly about it.

Reply:
I'm fine with just using `IsSet`. At least with `WithFluxes` it should be clearer what we're testing instead of `Independent`